metrics.py

```diff
@@ -114,9 +112,9 @@ def get_resource_metrics_for_executor_tasks(tasks):
 
     cpu_millis = (last_task_to_finish.end_total_cpu_jiffies -
       first_task_to_start.start_total_cpu_jiffies) * MILLIS_PER_JIFFY
-    total_transmit_idle_millis = (last_task_to_finish.end_network_transmit_idle_millis -
+    transmit_idle_millis = (last_task_to_finish.end_network_transmit_idle_millis -
       first_task_to_start.start_network_transmit_idle_millis)
-    network_metrics = NetworkMetrics(elapsed_millis, total_transmit_idle_millis)
+    network_metrics = NetworkMetrics(elapsed_millis, transmit_idle_millis)
```

Comment:
Can you pass these as named params (and same for all the other ones) to avoid future bugs?